dodiscover/ci/_script.py

```diff
@@ -0,0 +1,63 @@
+# type: ignore
+
+from math import frexp
+
+import numpy as np
+
+from dodiscover.ci import g_square_binary, g_square_discrete
+from dodiscover.ci.tests import testdata
+
+if __name__ == "__main__":
+
+    dm = np.array([testdata.bin_data]).reshape((5000, 5))
+    x = 0
+    y = 1
+
+    sets = [[], [2], [2, 3], [3, 4], [2, 3, 4]]
+    for idx in range(len(sets)):
+        print("x =", x, ", y =", y, ", s =", sets[idx], end="")
+        _, p = g_square_binary(dm, x, y, set(sets[idx]))
+        print(", p =", p, end="")
+        fr_p = frexp(p)
```

Comment:
I assume this code is breaking down the p value float into components so you can assert approximate equality.  Is this necessary?  Is this a place where we would see a lot of numeric variation across devices?

Reply:
Ah yeah. I think this can be deleted tbh. It was some debugging code.